odoo/addons/base/views/ir_ui_view_views.xml

```diff
@@ -43,6 +43,9 @@
                                     <field name="priority"/>
                                     <field name="name"/>
                                     <field name="xml_id"/>
+                                    <field name="active" invisible="1" />
+                                    <!-- Manually add active here. Otherwise that field will be readonly for the sub-list,
+                                    preventing saving changes done in the sub-form -->
```

Comment:
Is this a widget that changes the value or an onchange?

Reply:
a widget in the subview form

Reply:
```suggestion
                                    preventing saving changes done from widget in the sub-form-->
```